configs/immudb.ini.dist

```diff
@@ -1,12 +1,14 @@
-dir = .
+dir = ./immudata
 network = tcp
 address = 127.0.0.1
 port = 3322
-dbname = immudata
+dbname = immudb
```

Comment:
@mmeloni is this ok? i thought it should also be `immudata`

Reply:
@mmeloni i just saw you've added an intermediary directoy named `immudata`, so i suppose it's ok.

Reply:
Yes the point is that actually dbname is  the name of the db istance. The folder in which data is written is dir. Internally it uses dir+dbname to build the complete path of data folder. @andrei-lihu it's ok?